hadoop-ozone/dist/src/main/compose/ozone-hdfs/docker-compose.yaml

```
@@ -33,7 +33,7 @@ services:
          - ../..:/opt/ozone
       command: ["hdfs","datanode"]
       environment:
-         HADOOP_CLASSPATH: /opt/ozone/share/hadoop/ozoneplugin/*.jar
+         HADOOP_CLASSPATH: /opt/ozone/share/hadoop/ozone/lib/*.jar
```

Sorry I missed this earlier: this points to non-existent jars.

```suggestion
         HADOOP_CLASSPATH: /opt/ozone/share/ozone/lib/*.jar
```